MetaMask/metamask-extension
# add trezor HD path for ledger wallets

Fixes: https://github.com/MetaMask/metamask-extension/issues/10531

Explanation:  

Allows Trezor users to access their accounts from a Ledger

Manual testing steps:  

1. Load Trezor from mnemonic.
2. Load Ledger from mnemonic.
2. Connect to Trezor in Metamask and note the first 10 accounts.
3. Connect to Ledger in Metamask using Trezor from dropdown for HD path.

User should be able to access all accounts they had on Trezor from their ledger.
